src/controlCenter/widgets/volume/sinkInputRow.vala

```diff
@@ -56,7 +56,11 @@ namespace SwayNotificationCenter.Widgets {
             }
 
             if (show_per_app_label) {
-                label.set_text (this.sink_input.name);
+                if (this.sink_input.name.length >=10) {
+                    label.set_text(this.sink_input.name.substring(0, 7).concat("...")); 
+                } else {
+                    label.set_text (this.sink_input.name);
+                }
```

Comment:
This can be done internally through GTK with:
```vala
label.set_wrap (false);
label.set_single_line_mode (true);
label.set_width_chars (CONFIG_VARIABLE_HERE);
label.set_ellipsize (Pango.EllipsizeMode.END);
```

I'd also like it to be a config option, something like `"show-per-app-label-length": -1` where if the value is below 1, it becomes -1 (tells GTK to calculate the length automatically)